invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/InvitationResultItem.js

```diff
@@ -0,0 +1,151 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from "react";
+import { randomKey, roles, randomBool } from "../mock";
+import { DateTime } from "luxon";
+import {
+  Table,
+  Grid,
+  Container,
+  List,
+  Dropdown,
+  Icon,
+  Loader,
+} from "semantic-ui-react";
+import { Image } from "react-invenio-forms";
+import { ActionButtons } from "./InvitationActionButtons";
+import PropTypes from "prop-types";
+import { ErrorPopup } from "../components/ErrorPopup";
+
+const RoleContent = ({ title, description }) => (
+  <List>
+    <List.Item>
+      <List.Content>
+        <List.Header>{title}</List.Header>
+        <List.Description>{description}</List.Description>
+      </List.Content>
+    </List.Item>
+  </List>
+);
+
+const RoleSelection = ({ currentValue, onChange }) => {
+  const options = roles.map(({ title, description }, index) => ({
+    key: randomKey(index),
+    text: title,
+    value: title.toLowerCase(),
+    is_selected: currentValue === title,
+    content: <RoleContent title={title} description={description} />,
+  }));
+
+  return (
+    <Dropdown
+      selection
+      options={options}
+      value={currentValue}
+      onChange={onChange}
+    />
+  );
+};
+
+export class InvitationResultItem extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const {
+      invitation,
+      onRoleChange,
+      isLoading,
+      key,
+      error,
+      success,
+      currentRole,
+      onErrorClose,
+    } = this.props;
+    const { receiver, status, expires_at } = invitation;
+
+    const formattedTime = DateTime.fromISO(expires_at).toRelative();
+
+    return (
+      <Table.Row className="community-member-item" key={key}>
+        <Table.Cell>
+          <Grid textAlign="left" verticalAlign="middle">
+            <Grid.Column width={4}>
+              <Image
+                src={receiver.links.avatar}
+                fallbackSrc="/static/images/square-placeholder.png"
```

Comment:
to be removed when `react-invenio-forms` released?

Reply:
ahh yes, if it becomes redundant because of the new release then i will change it.